bugstats.php.dd

```diff
@@ -43,6 +43,6 @@ $(P <center>$(LINK2 https://issues.dlang.org/reports.cgi?product=D&datasets=NEW&
 Macros:
   TITLE=The D Bug Tracker
   PHP=<?php $0 ?>
-  BOOKTABLE = <center><table cellspacing=0 cellpadding=5 valign=top class=book><caption>$1</caption>$2</table></center>
+  BOOKTABLE = <center><table cellspacing=0 cellpadding=5 valign=top class="book"><caption>$1</caption>$2</table></center>
```

Comment:
Shouldn't stuff like the cellspacing, cellpadding, etc. also be moved to CSS?  Or are you thinking of doing this in a 3-stage process, first introducing the CSS references, then implementing the actual CSS design, and finally stripping out last hard-coded style elements?